src/ert/_c_wrappers/enkf/ert_workflow_list.py

```diff
@@ -162,6 +162,17 @@ def free(self):
     def __ne__(self, other):
         return not (self == other)
 
+    def __repr__(self):
+        job_dicts = [
+            {ConfigKeys.NAME: name, ConfigKeys.PATH: self.getJob(name).executable()}
+            for name in self.getJobNames()
+        ]
+        return (
+            "ErtWorkflowList(config_dict={"
+            f"{ConfigKeys.LOAD_WORKFLOW_JOB}: {job_dicts}, "
```

Comment:
should this one also have single quotes? hope i'm not bugging you too much about format stuff...

Reply:
No problem! Fixed